home.html

```diff
@@ -0,0 +1,205 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Penda Ledership Conference</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet"
+      integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+    <link rel="stylesheet" href="CSS/style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato|Poppins|Anton">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+  </head>
+
+  <body>
+    <header>
+      <div class="socials container-fluid bg-dark text-light d-md-flex d-none justify-content-end px-3 gap-4 px-5">
+        <a href="#"><img src="Images/facebook.png" alt="Facebook"></a>
+        <a href="#"><img src="Images/twitter.png" alt="twitter"></a>
+        <span>English</span>
+      </div>
```

Comment:
- [ ] Hey @c00p75, you did an awesome job on this project 👏. However, I have noticed that the items in the `top nav bar` is not aligned to the `right ` with those in the bottom nav bar. And compared to the design there are some items that are remaining. So, what do you think about fixing that? 🤔

### Your design 👇️
![Screenshot from 2022-11-26 02-04-40](https://user-images.githubusercontent.com/89228743/204068517-16197706-f472-4528-bdb1-075f7de5c58f.png)

### The original design 👇️
![Screenshot from 2022-11-26 02-04-00](https://user-images.githubusercontent.com/89228743/204068537-618abac9-1a47-4aa6-aad5-27c389676513.png)